README.md

```diff
@@ -738,14 +738,14 @@ into slave environment and set `DESTROY_DATABASE_ON_RESTART: 'False'`.
 
 After this, you can make changes to your replicant, but master and replicant will not
 be in sync anymore. This is useful if the replicant needs to take over a failover master.
-However it is recommended to take additional action, such as creating a backup from the
+However, it is recommended to take additional action, such as creating a backup from the
 slave so a dedicated master can be created again.
 
 #### Preventing replicant database destroy on restart
 
 You can optionally set `DESTROY_DATABASE_ON_RESTART: 'False'` after successful sync
 to prevent the database from being destroyed on restart. With this setting you can
-shut down your replicant and restart it later and it will continue to sync using the existing
+shut down your replicant and restart it later ,and it will continue to sync using the existing
```

Comment:
Comma should be 1 character to the left.

Reply:
Fixed in https://github.com/kartoza/docker-postgis/pull/370/commits/e3d347dcaad9d059c2277cbea7b136a2d4994c35